specs/block_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'simplecov'
+SimpleCov.start
+require_relative 'spec_helper'
+
+describe "Block" do
+  describe "initialize" do
+    it "is an instance of Block" do
+      start_date = Date.new(2018, 3, 7)
+      end_date = Date.new(2018, 3, 10)
+      block = Hotel::Block.new(start_date, end_date, 3)
+      block.must_be_kind_of Hotel::Block
+    end
+
+    it "will not accept more than 5 or less than 2 rooms" do
+      start_date = Date.new(2018, 3, 7)
+      end_date = Date.new(2018, 3, 10)
+      proc{Hotel::Block.new(start_date, end_date, 1)}.must_raise ArgumentError
+      proc{Hotel::Block.new(start_date, end_date, 6)}.must_raise ArgumentError
+    end
+
+    it "contains a blocked_rooms array" do
+      start_date = Date.new(2018, 3, 7)
+      end_date = Date.new(2018, 3, 10)
+      block = Hotel::Block.new(start_date, end_date, 3)
+      block.blocked_rooms.must_be_instance_of Array
+    end
+
+    it "gives the correct cost" do
+      start_date = Date.new(2018, 3, 7)
+      end_date = Date.new(2018, 3, 10)
+      block = Hotel::Block.new(start_date, end_date, 3)
+      block.full_cost.must_equal 1350.00
+      block.cost_per_room.must_equal 450.0
+    end
+  end
+
+  describe "reserve method" do
```

Comment:
You also need to test that it can't include a room in a block if the room has a reservation or at least that no room in a given block has a reservation in that time.